client/document/index.jsx

```diff
@@ -191,6 +191,7 @@ class Document extends Component {
 							src={ getBilmurUrl() }
 							data-provider="wordpress.com"
 							data-service="calypso"
+							data-customproperties={ `{"route": "${ sectionName }"}` }
```

Comment:
Should we call the new property `section`? Or something more specific, if necessary, `calypso_section` for example? This isn't really a route, after all, so one analyzing the RUM data later could be confused.

Reply:
Thank you, @tyxla!

We could use `section`, yes, but this custom property is already in use elsewhere, and we already have the graphs set up for it in the shared dashboard 😕 And to be fair, "section" would have probably been a better name there too... Wish I'd thought of that.

Do you think the naming mismatch is worth the added convenience and consistency of having it all match and already configured in the graphs?

Or, alternatively, is there something else we could pass here that would be closer to what a Calypso developer analysing the data would expect for "route"? As long as it's independent of any variables (e.g. the blog name), it should be useful. The idea behind this is to avoid the problem that URL path depends on blog name or other variables in a lot of cases, which makes it useless as an aggregating dimension to e.g. evaluate the overall performance of the home screen.

Reply:
Actually, what I can do is rename the graphs and the dimension's description to something like "Route / section". That should help!

Reply:
Also, I made a mistake; it's supposed to be `route_name` to match what's already there. Fixed in 47c4cefa6d97fa70ae1556b95825c961c69f7681.

Reply:
Rationale makes sense to me 👍 And you're right we can rename them for presentation purposes, so let's go with what you have.